src/coreclr/gc/gc.cpp

```diff
@@ -53701,6 +53708,10 @@ void PopulateDacVars(GcDacVars *gcDacVars)
         gcDacVars->dynamic_adaptation_mode = nullptr;
 #endif //DYNAMIC_HEAP_COUNT
     }
+    if (v6)
+    {
+        gcDacVars->gc_descriptors = (void*)&GCContractDescriptor;
```

Comment:
Nit: Should it be `gc_descriptor` (or `gc_contract_descriptor`) since there is just one?